app/src/organisms/Navigation/__tests__/NavigationMenu.test.tsx

```diff
@@ -9,19 +9,19 @@ import { useLights } from '../../Devices/hooks'
 import { RestartRobotConfirmationModal } from '../RestartRobotConfirmationModal'
 import { NavigationMenu } from '../NavigationMenu'
 
-import type { useHistory } from 'react-router-dom'
+import type { useNavigate } from 'react-router-dom'
```

Comment:
https://reactrouter.com/en/main/hooks/use-navigate#usenavigate


```suggestion
import type { NavigateFunction } from 'react-router-dom'
```